jgrapht-core/src/main/java/org/jgrapht/alg/flow/MaximumFlowAlgorithmBase.java

```diff
@@ -60,58 +61,85 @@
      * Default tolerance.
      */
     public static final double DEFAULT_EPSILON = 1e-9;
+    public final double EPSILON; // tolerance (DEFAULT_EPSILON or user-defined)
```

Comment:
CAPS naming-style is only used for the constant-fields, and is prohibited for the ordinary ones.